edb/server/protocol/edgeql_ext.pyx

```diff
@@ -96,6 +106,10 @@ async def handle_request(
         if variables is not None and not isinstance(variables, dict):
             raise TypeError('"variables" must be a JSON object')
 
+        # XXX: wait, should we...
```

Comment:
What's this comment in relation to?

Reply:
Honestly I don't remember. I deleted it.